### PR TITLE
feat: Polygon deployed

### PR DESCRIPTION
**Pull request by dekz**
## Description

```
137
        wmatic: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270',
```

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/contract-addresses/addresses.json [hunk 1]
```diff
@@ -244,5 +244,87 @@
...
+        "exchangeProxyLiquidityProviderSandbox": "0x4dd97080adf36103bd3db822f9d3c0e44890fd69",
+        "transformers": {
+            "wethTransformer": "0xe309d011cc6f189a3e8dcba85922715a019fed38",
+            "payTakerTransformer": "0x5ba7b9be86cda01cfbf56e0fb97184783be9dda1",
+            "affiliateFeeTransformer": "0xbed27284b42e5684e987169cf1da09c5d6c49fa8",
+            "fillQuoteTransformer": "0x76377186326848aeb369876027e12b64e2541ac5",
+            "positiveSlippageFeeTransformer": "0x4cd8f1c0df4d40fcc1e073845d5f6f4ed5cc8dab"
+        }
+    },
+    "80001": {
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/addresses.json:289]: prob not worth it, we don't even use the bsc testnet

File: packages/contract-addresses/addresses.json [hunk 2]
```diff
@@ -244,5 +244,87 @@
...
+        "erc20BridgeProxy": "0x0000000000000000000000000000000000000000",
+        "erc20BridgeSampler": "0x0000000000000000000000000000000000000000",
+        "chaiBridge": "0x0000000000000000000000000000000000000000",
+        "dydxBridge": "0x0000000000000000000000000000000000000000",
+        "godsUnchainedValidator": "0x0000000000000000000000000000000000000000",
+        "broker": "0x0000000000000000000000000000000000000000",
+        "chainlinkStopLimit": "0x0000000000000000000000000000000000000000",
+        "maximumGasPrice": "0x0000000000000000000000000000000000000000",
+        "dexForwarderBridge": "0x0000000000000000000000000000000000000000",
+        "exchangeProxyGovernor": "0x4d3e56c56a55d23fc7aa9a9ffad61631cf7d1ae6",
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/addresses.json:276]: so one annoying thing is we usually use etherscan to interact the governor. maybe we should deploy a flattened version of the governor so we can interact with it on vigil

File: packages/contract-addresses/addresses.json [hunk 3]
```diff
@@ -244,5 +244,87 @@
...
+        "exchangeProxy": "0xdef1c0ded9bec7f1a1670819833240f027b25eff",
+        "exchangeProxyTransformerDeployer": "0xe6d9207df11c55bce2f7a189ae95e3222d5484d3",
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/addresses.json:278]: maybe same for this

**Review by dorothy-zbornak — APPROVED** [review]

File: packages/contract-addresses/addresses.json (hunk 2 above)
Comment (dorothy-zbornak) [reply packages/contract-addresses/addresses.json:276]: nvm  we'll figure it out

**Comment by kimpers** [comment]
I think we can close this in favor of #240 which is based on this branch

**Pull request closed without merging**